lib/guard/yardstick.rb

```diff
@@ -98,5 +99,16 @@ def smart_path(path)
         path
       end
     end
+
+    # Merge further options with yardstick config file.
+    #
+    # @api private
+    # @return [Hash] Hash of options for yardstick measurement
+    def yardstick_config(extra_options = {})
+      return options unless options[:config]
+
+      yardstick_options = YAML.load_file(options[:config])
```

Comment:
There is a open PR on yardstick that would make this obsolete. https://github.com/dkubb/yardstick/pull/50